src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -268,6 +268,17 @@ public void addTeachingAssistant(final String taGroup, final String taName) {
         assertThat(ta.getId()).as("Teaching assistant has been created").isNotNull();
     }
 
+    public Lecture createCourseWithLecture() {
+        Course course = ModelFactory.generateCourse(null, pastTimestamp, futureFutureTimestamp, new HashSet<>(), "tumuser", "tutor", "instructor");
+
+        Lecture lecture = new Lecture();
+        lecture.setDescription("Test Lecture");
+        lecture.setCourse(course);
+        courseRepo.save(course);
+        // lectureRepo.save(lecture);
```

Comment:
The method name would suggest that the lecture itself also gets saved. I also saw that in most use cases, the lecture is saved after this method is called. Maybe it makes sense to have a boolean flag for this where the default is that the lecture also gets saved?